src/core.ts

```
@@ -173,18 +366,73 @@
+      case OutputFormat.CSV:
+        await writeFile(
+          nextFileNameString,
+          stringify(currentResults, {
+            header: true,
+            columns: Object.keys(currentResults[0] || {}),
+          }),
+        );
+        break;
+
+      case OutputFormat.MARKDOWN:
+        let mdContent = "# Crawled Content\n\n";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)